plugins/system/schedulerunner/schedulerunner.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * @package         Joomla.Plugin
+ * @subpackage      System.ScheduleRunner
+ *
+ * @copyright   (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license         GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+// Restrict direct access
+defined('_JEXEC') or die;
+
+use Assert\AssertionFailedException;
+use Joomla\CMS\Application\CMSApplication;
+use Joomla\CMS\Component\ComponentHelper;
+use Joomla\CMS\Plugin\CMSPlugin;
+use Joomla\Component\Scheduler\Administrator\Scheduler\Scheduler;
+use Joomla\Event\DispatcherInterface;
+use Joomla\Event\Event;
+use Joomla\Event\SubscriberInterface;
+
+/**
+ * The plugin class for Plg_System_Schedulerunner.
+ *
+ * @since __DEPLOY_VERSION__
+ */
+class PlgSystemSchedulerunner extends CMSPlugin implements SubscriberInterface
+{
+	/**
+	 * @var  CMSApplication
+	 * @since  __DEPLOY_VERSION__
+	 */
+	protected $app;
+
+
+	/**
+	 * Override parent constructor.
+	 * Prevents the plugin from attaching to the subject if conditions are not met.
+	 *
+	 * @param   DispatcherInterface  $subject  The object to observe
+	 * @param   array                $config   An optional associative array of configuration settings.
+	 *
+	 * @since __DEPLOY_VERSION__
+	 */
+	public function __construct(&$subject, $config = [])
+	{
+		// Make sure com_scheduler is installed and enabled
+		if (!ComponentHelper::isEnabled('com_scheduler'))
+		{
+			return;
+		}
+
+		parent::__construct($subject, $config);
+	}
+
+	/**
+	 * Returns event subscriptions
+	 *
+	 * @return string[]
+	 *
+	 * @since __DEPLOY_VERSION__
+	 */
+	public static function getSubscribedEvents(): array
+	{
+		return [
+			'onBeforeRender' => ['registerRunner', PHP_INT_MAX]
```

Comment:
Nice try, but please use the allowed definitions
```suggestion
			'onBeforeRender' => ['registerRunner', \Joomla\Event\Priority::MAX]
```

Reply:
Neat, will do! 😃